rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -304,6 +304,9 @@ usb_handler_thread::usb_handler_thread()
 
 		// EA Active 2 dongle for connecting wristbands & legband
 		check_device(0x21A4, 0xAC27, 0xAC27, "EA Active 2 Dongle");
+
+		// Tony Hawk RIDE Skateboard
+		check_device(0x12BA, 0x0400, 0x0400, "Tony Hawk RIDE Skateboard Controller");
```

Comment:
This isn't neccesary, right? The games don't use usbd.

Reply:
Correct, this was added in the beggining for testing